component/service/src/main/java/org/exoplatform/social/rest/impl/space/SpaceRestResourcesV1.java

```diff
@@ -182,6 +183,11 @@ public Response getSpaces(@Context UriInfo uriInfo,
         space.setHasBindings(spaceBindingService.isBoundSpace(space.getId()));
         
         SpaceEntity spaceInfo = EntityBuilder.buildEntityFromSpace(space, authenticatedUser, uriInfo.getPath(), expand);
+        List<UserSpaceBinding> userSpaceBinding = spaceBindingService.findUserSpaceBindingsBySpace(space.getId(), authenticatedUser);
+        spaceInfo.setIsUserBound(false);
```

Comment:
To know if user is bound from group or not, we seems to have only one method :
`spaceBindingService.countUserBindings(space.getId(), userId) > 0`
@rdenarie , do you confirm ?

Reply:
Yes I confirm. In fact, we add it in webui utils file : 
https://github.com/Meeds-io/social/blob/develop/component/webui/src/main/java/org/exoplatform/social/webui/Utils.java#L889
It could be usefull to add it in GroupSpaceBindingService if needed in different location

Reply:
Thank you @rdenarie .

@Rdidmohamedelhadi I let you change this into : 

`spaceInfo.setIsUserBound(groupSpaceBindingService.countUserBindings(spaceId, authenticatedUser) > 0)`
(we should avoid adding WebUI artifact dependency)

Just another concern, I think this information is heavy when requesting the list of spaces, so we should retrieve this field only when `expand` contains `"all"`, @ahamdi WDYT ?